cmd/client.cc

```diff
@@ -117,14 +117,14 @@ int main() {
   uint32_t *msg_num = (uint32_t *)&data_buf;
 
   while (true) {
-    for (int i = 0; i < 12; i++) {
+    for (int i = 0; i < 10; i++) {
```

Comment:
Bit of a magic number? Would be nice if it had a name.

Reply:
I'll add a config/env var for it. It's basically what you want to use for testing.  Same thing for the sleep_for.